### PR TITLE
feat(results.js, script.js): created a function that tests filter pai…

### LINKED ISSUE #52: [Feat]: create a function that tests filtered pairing percentage accuracy
### Is your feature request related to a problem? Please describe.

we need to evaluate how accurate our generating is based on filters.

### Describe the solution you'd like.

create a function that tests how accurate the pairing is based on if location and department are toggled on.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by vlomeli**
…ring accuracy;

**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Changes
_List Changes Introduced by this PR_
1. added two variables to session storage
2. used the variables in calculatePairAccuracy function that makes sure that the if filters are on the value for that filter is === on both interns that are paired.
3. added new file for accuracy.js
4. changed text for filter buttons and pair btn.

## Purpose
this just shows us the accuracy of our randomizer based on the filters selected.

## Approach
This approach allows us to reuse the variables used in script.js file in results.js. It is just conditional statements that check the accuracy of the filters.

## Pre-Testing TODOs
none

## Testing Steps
run the randomizer with people in the selected pool and check the console.

## Learning
I learned how I could use variables with session storage.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #52

### CODE REVIEW

File: results.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import * as generate_CSV from "./src/util/generate_csv.js"; 
+import * as Accuracy from "./src/util/accuracy.js";
```
Comment (joavelar) [inline results.js:2]: Nice, thanks for being consistent.

File: src/util/accuracy.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+
+
+ export function calculatePairAccuracy(internPairs) {
+    let isPairedByLocation = JSON.parse(sessionStorage.getItem('isPairedByLocation'));
```
Comment (joavelar) [inline src/util/accuracy.js:4]: I think you may be able to export and import variables rather than using sessionStorage.

**Review by joavelar — APPROVED** [review]
Good job! I appreciate you separating this into its own file.

**Review by MasellisL — APPROVED** [review]
Looks good to me! besides what Jose mentioned!!!

File: src/util/accuracy.js (hunk 2 above)
Comment (vlomeli) [reply src/util/accuracy.js:4]: After doing some quick research I found that using session storage is the best way to go about this to keep the variables most up to date.

**Pull request merged**